week-4/implement/count.test.js

```diff
@@ -15,3 +15,31 @@
 // And a character char that does not exist within the case-sensitive str,
 // When the function is called with these inputs,
 // Then it should return 0, indicating that no occurrences of the char were found in the case-sensitive str.
+
+
+function countChar(str, char) {
+    let count = 0;
+    for (let i = 0; i < str.length; i++) {
+        if (str.slice(i, i + 1) === char) {
+            count++;
+        }
+    }
+    return count;
+}
```

Comment:
A nice use of the for loop structure in your function. Also a nice use of the `slice()` method to get the character out of a word.